File: fix_imports.py
import re

files_to_fix = [
    './app/src/main/java/com/example/ui/components/HomeFeatureCards.kt',
    './app/src/main/java/com/example/ui/components/QuranPlannerTabContent.kt',
    './app/src/main/java/com/example/ui/components/KhatmPlannerSheet.kt'
]

for filepath in files_to_fix:
    with open(filepath, 'r') as f:
        content = f.read()

    # Change back to Color.semanticPrimaryAccent
    content = content.replace("androidx.compose.ui.graphics.Color.semanticPrimaryAccent", "Color.semanticPrimaryAccent")
    
    # Add import if missing
    if "import com.example.ui.theme.semanticPrimaryAccent" not in content:
        # insert after package
        content = re.sub(r'(package [^\n]+\n)', r'\1\nimport com.example.ui.theme.semanticPrimaryAccent\nimport androidx.compose.ui.graphics.Color\n', content)
        
    with open(filepath, 'w') as f:
        f.write(content)

File: fix_imports.js
const fs = require('fs');
['app/src/main/java/com/example/ui/screens/NamesOfAllahScreen.kt', 'app/src/main/java/com/example/ui/screens/QuranScreen.kt'].forEach(file => {
    let content = fs.readFileSync(file, 'utf-8');
    if (content.startsWith('import com.example.ui.theme.semanticDockBorder\npackage com.example.ui.screens')) {
        content = content.replace('import com.example.ui.theme.semanticDockBorder\npackage com.example.ui.screens', 'package com.example.ui.screens\nimport com.example.ui.theme.semanticDockBorder');
        fs.writeFileSync(file, content);
    }
});
